### ISSUE TITLE
Planets kyanite pool not regenerated

### ISSUE THREAD

**Issue by Rider868** [onikamigames/starshipgame#471]
There seems to be an issue with my planets kyanite pool not regenerating after 24 hours. planet ID 28976 would appreciate your help. I checked it the next scheduled update was supposed to be on Jan 1st 7pm utc  
Thank you

**Comment by waterlenz** [comment]
yes - planets 3, 8, 4105, 8202, 12296, 12291, 12299, and 12288 are all more than 24hours overdue!!!!

**Comment by Rider868** [comment]
Someone in discord posted a solution 

eyes IMPORTANT INFO: Regarding planet pools apparently not resetting eyes 

 I have a theory about why planet pools APPEAR to not be resetting! Basically, for the game to reset every planet pool on the blockchain at midnight UTC, there would have to be a scheduled task to trigger an update to all 30,000 planets in the game - this would be incredibly intensive and inefficient on the blockchain, so I don't think they are doing this. Also, this would become more inefficient for every spacestation and additional 10,000 planets added to the game. So, something else would have to trigger the reset. The only other thing that could trigger this is somebody mining the planet (then if they are the first person to mine it after the reset, the pool data would reset to max, less the amount mined by this first ship of the day - this is way more efficient, but means the blockchain does not always show the correct data). This is actually how many timer-based games work.
So basically, the pools ARE technically resetting (the code knows when a reset has passed and when a ship visits), but because everybody thinks the planet is empty, nobody is going to mine it, so the count never gets updated - it's pretty hard to explain, but as a dev, I'm 99.9% sure this is the case.
I'll see if I can update my planet tool to take this in to account tomorrow.
